code_generator/GdkCodeGenerator/Templates/UnityReaderWriterGenerator.tt

```diff
@@ -10,10 +10,13 @@
 #>
 <#= generatedHeader #>
 
+using System;
```

Comment:
Optional - Nit: add the usings as conditional per #evtDetails>0? Or use full names?

Reply:
Done, fully qualified the names.